ports/strikeysisters/Strikey Sisters.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+if [ -f "${controlfolder}/libgl${CFWNAME}.txt" ]; then 
```

Comment:
This needs to go after line 60 where you cd into `$GAMEDIR` 😄

Reply:
so get_controls is also after CD gamedir?

Reply:
okies updated (I hope it is right)